tools/vipps-psp-v2-api-postman-collection.json

```diff
@@ -133,6 +141,10 @@
 								"value": "{{Ocp-Apim-Subscription-Key}}",
 								"type": "text"
 							},
+							{
+								"key": "Merchant-Serial-Number",
+								"value": "{{merchantSerialNumber}}"
+							},
```

Comment:
I don't really know enough about this api, maybe ask towsend, but from what I can see at least, there would be no merchantSerialNumber to provide here, since you are creating the merchant that should get one.

Reply:
I think you're right. I've removed them from the whole merchant signup section.